packages/icons/bin/svgo-jsx-index.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from "fs/promises";
+import * as path from "path";
+
+(async () => {
+  try {
+    const list = await fs.readdir(path.join(process.cwd(), "./src/gen"));
```

Comment:
we don't really need a special folder I think :)

Reply:
it's useful when we delete svg and want to synchronize list of generated icons

Reply:
got it

Reply:
wait, then you can delete whatever svg was actually deleted, you know a ll svgs

Reply:
ah no, if its deleted you don't know if the other one exists and is not manual, yeah we need a generated folder